inst/rmarkdown/templates/vetiver_dashboard/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,99 @@
+---
+title: "Monitor model"
+output: 
+  vetiver::vetiver_dashboard:
+    pins:
+      board: !expr pins::board_local()
+      name: 'seattle_rf'
+      version: NULL
+    storyboard: true
+    source_code: 'https://vetiver.rstudio.com'
+    display_pins: true
+---
+
+```{r setup, include = FALSE}
+# packages needed for monitoring
+library(vetiver)
+library(pins)
+library(plotly)
+
+# packages needed for prediction
+library(parsnip)
+library(workflows)
+
+knitr::opts_chunk$set(echo = FALSE)
+pins <- get_vetiver_dashboard_pins()
```

Comment:
Ok I think I have missed something important. 
```r
pins <- get_vetiver_dashboard_pins()
```
This will work well during knitting only and not interactively as the **knitr** option `knitr::opts_knit$get("vetiver_dashboard.pins")` won't be populated by the IDE - it will only be populated when the document is knitted and `vetiver::vetiver_dashboard()` is effectively evaluated. 

This may not be what you wanted considering that is was working as expected for you using `params`. Am I right ? 

Sorry for the oversight on this. The RStudio IDE in fact do a specific processing of the YAML `params` fields to make it available in the global environment for interactive execution of chunk. 

So we probably need to reconsider the current solution, aren't we ?